docker-compose.yml

```diff
@@ -101,6 +101,28 @@ services:
       - 5432:5432
     volumes:
       - /var/lib/postgresql/data
+  rbac:
+    image: quay.io/cloudservices/rbac:latest
+    restart: on-failure
+    environment:
+      - REDIS_PORT=6379
+      - REDIS_HOST=redis
+      - DATABASE_HOST=db
+      - DATABASE_PORT=5432
+      - DATABASE_NAME=insights
+      - DATABASE_USER=insights
+      - DATABASE_PASSWORD=insights
+      - API_PATH_PREFIX=/api/rbac
+      - DEVELOPMENT=True
+    depends_on:
+      - db
+      - redis
+    links:
+      - db
+      - redis
+      - rails
```

Comment:
Can we also have a dependency for `rbac` into the `rails`, `sidekiq` and maybe `inventory-consumer`?

Reply:
Dependence and linking are both necessary for these 3

Reply:
With the current chanages, applying dependence and linking results in the following error after running `podman-compose` command:

```
podman-compose down
['podman', '--version', '']
using podman version: 4.0.1
Traceback (most recent call last):
  File "/usr/bin/podman-compose", line 33, in <module>
    sys.exit(load_entry_point('podman-compose==1.0.3', 'console_scripts', 'podman-compose')())
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 1775, in main
    podman_compose.run()
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 1022, in run
    self._parse_compose_file()
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 1102, in _parse_compose_file
    flat_deps(services, with_extends=True)
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 803, in flat_deps
    rec_deps(services, name)
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 777, in rec_deps
    new_deps = rec_deps(services, dep_name, start_point)
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 777, in rec_deps
    new_deps = rec_deps(services, dep_name, start_point)
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 777, in rec_deps
    new_deps = rec_deps(services, dep_name, start_point)
  [Previous line repeated 991 more times]
  File "/usr/lib/python3.10/site-packages/podman_compose.py", line 767, in rec_deps
    for dep_name in deps.copy():
RecursionError: maximum recursion depth exceeded while calling a Python object
```